opds.py

```diff
@@ -1196,3 +1186,93 @@ def create_entry(self, work, lane_link):
                 "I know about this work but can offer no way of fulfilling it."
             )
 
+# Mock annotators for use in unit tests.
```

Comment:
I moved these into opds.py from a test class because TestAnnotator is now used by multiple test classes.